src/robot.cpp

```diff
@@ -610,7 +639,7 @@ void Robot::executeManipulationPath()
   {
     bool success = false;
     group_it->second->setStartStateToCurrentState();
-    group_it->second->setRandomTarget();
+    //group_it->second->setRandomTarget();
```

Comment:
if not used, then just remove it

Reply:
Not sure why It was there in first place